projects/Mallard/src/navigation/navigators/article.tsx

```diff
@@ -195,9 +198,10 @@ const createArticleNavigator = (
             front,
             () => animatedValue,
         ),
-        [routeNames.Article]: supportsTransparentCards()
-            ? wrapInSlideCard(article, () => animatedValue)
-            : wrapInBasicCard(article, () => new Animated.Value(1)),
+        [routeNames.Article]:
+            !supportsAnimation() || !supportsTransparentCards()
```

Comment:
```
supportsAnimation() || supportsTransparentCards()
? wrapInSlideCard(article, () => animatedValue)
: wrapInBasicCard(article, () => new Animated.Value(1)),
```

Reads a little better so that both the predicates aren't both negated.

Reply:
Supports Transparent Cards = all but Android
supportsAnimation = all but iOS9

Your suggestion would mean iOS9 would pass and use Slide Card instead of Basic.

I hate it too